src/pfs/pfs.proto

```diff
@@ -300,6 +300,19 @@ message CreateBranchRequest {
   bool new_commit_set = 5; // overrides the default behavior of using the same CommitSet as 'head'
 }
 
+message SearchForFileInBranchRequest {
+  Commit start = 1;
+  string file_path = 2;
+  int32 limit = 3;
+  google.protobuf.Duration timeout = 4;
+}
+
+message SearchForFileInBranchResponse {
+  repeated Commit found_commits = 1;
+  Commit last_searched_commit = 2;
```

Comment:
Can this just be a Commit ID instead of the whole Commit?  Is there any benefit to sending over the whole thing?
The downsides I can see are:
- Using more bandwidth
- Being mistaken for a relevant commit

Reply:
Returning the commit is more convenient for the client since they can use the reference to begin a new search without having to make another API call to get the commit from the ID.

Reply:
Yeah that makes sense.  I got Commit confused with CommitInfo.